working/macros/feature-specification.md

```diff
@@ -246,6 +246,41 @@ introspection APIs.
 These macros can fully introspect on any type reachable from the declarations
 they annotate, including introspecting on members of classes, etc.
 
+## Macro Instantiation
+
+Macros should be able to be able to be instantiated using only the libraries
+that are available to the macro definition library. This allows us to compile an
+app (or spawn an isolate) that can run macro applications for that macro using
+only its own dependencies.
```

Comment:
Who performs this instantiation? Where do they get the arguments from?

Reply:
Some of the other wording below starts to address this - I don't want to get overly specific as this could be implementation specific? For now its just "the compiler".